app/sim_test.go

```diff
@@ -229,6 +231,7 @@ func TestAppImportExport(t *testing.T) {
 
 		{app.keys[subspacestypes.StoreKey], newApp.keys[subspacestypes.StoreKey], [][]byte{}},
 		{app.keys[profilestypes.StoreKey], newApp.keys[profilestypes.StoreKey], [][]byte{}},
+		{app.keys[relationshipstypes.StoreKey], newApp.keys[relationshipstypes.StoreKey], [][]byte{}},
```

Comment:
This is to fix @bragaz comment on the relationships PR which I didn't address due to the automerge